integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -12,6 +12,9 @@ kubernetes_dir=$(dirname "$(readlink -f "$0")")
 cidir="${kubernetes_dir}/../../.ci/"
 source "${cidir}/lib.sh"
 
+# Using trap to ensure the cleanup occurs when the script exists.
+trap './cleanup_env.sh' EXIT
```

Comment:
I wonder if the `./` path depends on what `pwd` is when the trap fires?
Might be safer to encode the full `${cidir}/cleanup_env.sh` here (I think that is the correct full path...)

Reply:
I think EXIT will be enough - you might want to check if we need QUIT and KILL as well though.

Reply:
yep, you're right~~~ all failed CI are telling me to use full path.😑